packages/router/src/config.ts

```diff
@@ -24,6 +25,43 @@ import {
 } from "@connext/nxtp-utils";
 import { config as dotenvConfig } from "dotenv";
 
+// TODO: Should we be caching chain data locally? Like, request at most once per day, or once per week?
+// Maybe there could be a versioning system of this remote config, and we can use that to check whether our
+// local cache is up to date?
+const CHAIN_DATA = new Promise<Map<string, any>>((resolve) => {
+  // TODO: Using raw github api is not a stable solution.
```

Comment:
Why not? I think this is a fine way to do it.

Reply:
Okay - I was worried that relying on github to host adds the constraint that, if github stops working, our routers stop working. Github probs not gonna stop working lol, but if it does, would suck.

I'll update this to be a TODO with the above statement regarding caching the remote config, and checking for updates. If github is randomly down, we can just revert to cached.

Reply:
Should probably actually go ahead and do that in this PR...

Reply:
We should be able to default to something if it's not working.

Reply:
Actually I'm not sure. This should only happen on startup anyways so I think it's okay to crash the router if this doesn't work.